egs/librispeech/ASR/finetune_hubert_transducer/conformer.py

```diff
@@ -0,0 +1,1848 @@
+#!/usr/bin/env python3
```

Comment:
Could you replace it with a symlink to pruned_transducer_stateless5/conformer.py?

Also for the other files that you copied directly, e.g., `joiner.py`.

(I see that you have already done this for asr_datamodule.py)